### ISSUE TITLE
About training set and test set

### ISSUE THREAD

**Issue by NNNNAI** [EndlessSora/DeeperForensics-1.0#9]
Do you use the entire DeeperForensics_1.0\source_videos as the training set, so when generating the swapped dataset, the model has actually been trained on all source images. The whole model is not subject agnostic. Is it true?

**Comment by EndlessSora** [comment]
Not sure about your question. You mean is the DF-VAE trained on all the source images to generate all the fake videos?

**Comment by NNNNAI** [comment]
十分抱歉，可能是因为我的英文水平有限。您说的对，我想问的就是DF-VAE是否使用整个DeeperForensics_1.0\source_videos的所有source image来训练。其实我疑惑的就是比如fake video 如DeeperForensics_1.0\manipulated_videos\end_to_end_mix_2_distortions010_W006.mp4，DF-VAE的训练集是否已经包含W006。

**Comment by EndlessSora** [comment]
Yes, you're correct. The training set of your specified example contains W006. You know, for the manipulation stage, it is not necessary to split a separate test set for popular off-the-shelf face swapping methods, since the source and target videos are indeed the ones we need to manipulate. Subject-agnostic face swapping is an interesting research topic in the future. :-)

**Comment by NNNNAI** [comment]
Thanks for your help~~~ : )
